phase1-coordinator/src/coordinator.rs

```diff
@@ -1447,7 +1458,6 @@ where
     ///
     /// On failure, this function returns a `CoordinatorError`.
     ///
-    #[inline]
```

Comment:
This line has nothing to do with round restart. It'll be nice to revisit our inline policy and to submit a separate PR deleting all inlines which are not justified by the benchmarks. Inlines are not always good for performance, but they always increase the compile times. @apruden2008 do you think it worth the time doing something with inlines?

Reply:
And sometimes they even decrease performance! A number of posts about that on the Rust Reddit I've seen recently.